### PR TITLE
Update loan type

### PR DESCRIPTION
**Pull request by ros3bud**
- Updated type for loan_amount from number to range
- Removed parseNumber method and calls, because all fields utilizing that method are already numbers
- Added test for range in loan_amount

### CODE REVIEW

**Comment by ros3bud** [comment]
@cgrayson, LGTY?

File: spec/inbound_spec.coffee [hunk 1]
```diff
@@ -278,7 +281,7 @@ body = (lead = {}) ->
     <est_prop_val>125500</est_prop_val>
     <borrow_amt>125000</borrow_amt>
     <veteran_yn>#{lead.is_military ? 'y'}</veteran_yn>
-    <loan_amount>100000</loan_amount>
+    <loan_amount>#{lead.loan_amount}</loan_amount>
```
Comment (cgrayson) [inline spec/inbound_spec.coffee]: Somewhat minor, but: if this had a default for when no `lead.loan_amount` is specified, then those additions of `loan_amount: 10000` above wouldn't be needed.

**Comment by cgrayson** [comment]
Also noticed (from the [travis build output](https://travis-ci.org/activeprospect/leadconduit-integration-homesdotcom/builds/71813020)) that this module could use the `sudo: false` treatment in `.travis.yml`.

Otherwise, LGTM.

File: spec/inbound_spec.coffee (hunk 1 above)
Comment (ros3bud) [inline spec/inbound_spec.coffee]: I see what you mean, thanks!

**Pull request closed without merging**